metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -1174,6 +1184,7 @@ class Compilers(
       debugItem.setSelectionStart(selection)
     }
 
+    debugItem.setDetail(item.getDetail())
```

Comment:
It seems that since some version we can add detail, which wasn't possible before.